fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBRecordStore.java

```diff
@@ -3854,6 +3928,10 @@ public Builder setMetaDataProvider(@Nullable RecordMetaDataProvider metaDataProv
             return this;
         }
 
+        public void setMetaDataPreloadFuture(@Nonnull CompletableFuture<Void> metaDataPreloadFuture) {
```

Comment:
If we keep this method, it should return this `Builder` instead of `void`.

Reply:
dropped.